api_app/db/repositories/shared_services.py

```diff
@@ -49,7 +50,11 @@ def get_deployed_shared_service_by_id(self, shared_service_id: str, operations_r
         return shared_service
 
     def get_shared_service_spec_params(self):
-        return self.get_resource_base_spec_params()
+        params = self.get_resource_base_spec_params()
+        params.update({
+            "azure_location": config.RESOURCE_LOCATION,
+        })
+        return params
```

Comment:
So the thing is, when I was building the shared service API, my intention was to pass through azure_location (however for it to work this change was missing). 
Now in this PR we seem to want to not do that for shared services, and instead use the core resource group. 
I think azure_location is still used in shared service template, so if I remove it, I need to remove it there as well. In my mind it would be better to keep things consistent in this PR and remove references to azure_location in a separate PR.

Reply:
For the sake of getting this PR through, I have removed this change. I will remove the remaining references in a follow-up PR.